Taskfile.yml

```diff
@@ -802,6 +823,8 @@ tasks:
       - az-login
     cmds:
       - "az group delete --name {{.RESOURCE_GROUP_NAME}} -y"
+      - "{{.SCRIPTS_ROOT}}/delete-role-assignment.sh -d {{.AKS_WORKLOAD_IDENTITY_PATH}}"
+      - "rm -rf {{.AKS_WORKLOAD_IDENTITY_PATH}}"
```

Comment:
Do we need `-rf` here? The combination can be dangerous if `AKS_WORKLOAD_IDENTITY_PATH` happens to get an odd value.

Reply:
Unfortunately yes I think we do need `-rf` here. `-r` because there's 2 levels of subfolder to remove and `-f` because otherwise it'll prompt which doesn't work in unattended scenarios.

I did some digging though and realized that now rm's default is:
>       --preserve-root[=all]  do not remove '/' (default);
>                              with 'all', reject any command line argument
>                              on a separate device from its parent

Which means actually, you can't blat your whole system with this anymore so it's much safer than it used to be. You _can_ still accidentally delete directories you didn't mean to, but in this case I think we're pretty safe because we've hardcoded it to a specific variable `.AKS_WORKLOAD_IDENTITY_PATH` and this path won't ever be unset unless we delete that variable.

Reply:
and if the variable is just deleted, we'll get this:

> rm: missing operand
> Try 'rm --help' for more information.